lib/spack/llnl/util/tty/color.py

```diff
@@ -161,7 +161,7 @@ def _err_check(result, func, args):
             )
             # Use conout$ here to handle a redirectired stdout/get active console associated
             # with spack
-            with open(r"\\.\CONOUT$", "w") as conout:
+            with open(r"\\.\CONOUT$", "w", encoding="utf-8") as conout:
```

Comment:
@johnwparent Can you check if this one is correct, or needs an exception / some other encoding?